src/specklepy/serialization/base_object_serializer.py

```diff
@@ -222,7 +224,7 @@ def traverse_value(self, obj: Any, detach: bool = False) -> Any:
         if isinstance(obj, Enum):
             return obj.value
 
-        elif isinstance(obj, (list, tuple, set)):
+        elif isinstance(obj, list | tuple | set):
```

Comment:
Are we going to use this notation or `from typing import List, Tuple, Set`? There is an inconsistency.

Reply:
Good point

Reply:
wait, this is not an issue, the isinstance check only works on the `list` object, not on the `List` type. That is a runtime check, not a typechecking thing